app/src/main/res/layout/activity_profile.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <ImageView
+        android:id="@+id/login_background"
```

Comment:
The id should be activity_profile_image_background ([component/activity]_[widget type]_[context])

Reply:
Moreover if this is a common background for the login activity, it may be better to use a common style in values/styles.xml (look at the documetation)